packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -401,6 +462,8 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
                   );
                 }
               }
+
+              await handleVisitAttributes(visitAttributes, response.data.uuid);
```

Comment:
`handleVisitAttributes()` never throws, so this will always continue as if the requests were successful, which denies the user the ability to actually fix the visit attributes...

Reply:
Also, since this bit is using rxjs, it's probably better to leave the function as non-async and add the call to `handleVisitAttributes()` as part of the rxjs chain, e.g.,

```ts
subscribe((response) => /* omitted logic */ response.data.uuid)
	.pipe(first())
  .subscribe((visitUuid) => from(handleVisitAttributes(visitAttributes, response.data.uuid)))
  .pipe(first())
  .subscribe(() => {
    mutateCurrentVisit();
    mutateVisits();
    closeWorkspace();
  });
```